manifests/yum/repo/local_os_updates.pp

```diff
@@ -1,77 +1,95 @@
-# @summary Configure yum to use a (simp-managed) OS Updates repository
+# @summary Configure yum to use a (SIMP-managed) OS Updates repository for
+#   network-isolated environments.
 #
-# Generally, this is used by the ISO installation.
+# Generally, this is used by the ISO installation's SIMP server and agents.
```

Comment:
Actually, the SIMP server uses the /etc/yum.repos.d/simp_filesystem.repo because it is more efficient and disables the repos used by the agents.  simp config disables for the SIMP server.